doc/adr/0004-global-snapshot-trigger.md

```diff
@@ -0,0 +1,23 @@
+# 4. Global snapshot trigger
+
+Date: 2022-01-05
+
+## Status
+
+Proposed
+
+## Context
+
+Global snapshot must have a deterministic trigger so then snapshots created on
+different nodes should converge (contains the same data) and pass the majority algorithm.
```

Comment:
See convo above, but tuning height diff and dag dimensions will help optimize redownload size if needed, in lieu of tuning the reputation model and optimizing facilitator/tip selection

Reply:
it works if all the data are known on each layer (especially tips)

Reply:
Exactly